src/components/MovieCard/MovieCard.js

```diff
@@ -3,20 +3,8 @@ import './MovieCard.css'
 
 function MovieCard(props) {
   return (
-    <img className='movie-card' src={props.poster} alt='A poster of the movie' onClick={() => props.findMovie(props.id)} ></img>
+    <img className='front-movie-card' src={props.poster} alt='A poster of the movie' onClick={() => props.findMovie(props.id)}></img>
```

Comment:
Love that you remembered alt text here - consider interpolating the movie name or something more descriptive